SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -232,6 +232,8 @@ const VirtualWorkerId INVALID_VIRTUAL_WORKER_ID = 0;
 const ActorLockToken INVALID_ACTOR_LOCK_TOKEN = 0;
 const FString INVALID_WORKER_NAME = TEXT("");
 
+static const FName DefaultLayer = FName(TEXT("UnrealWorker"));
```

Comment:
Wonder if it would be neat to use the value from SpatialEngineConstants?

Reply:
I'm going to in the PR when I do the pass to remove ActorGroups.